tests/basic_relay.go

```diff
@@ -298,29 +217,17 @@ func sendBasicTeleporterMessage(
 	destination interfaces.SubnetTestInfo,
 	fundedKey *ecdsa.PrivateKey,
 	fundedAddress common.Address,
-) (teleportermessenger.TeleporterMessage, ids.ID) {
-	log.Info("Packing Teleporter message")
-	teleporterMessage := teleportermessenger.TeleporterMessage{
-		MessageNonce:            big.NewInt(1),
```

Comment:
This was an existing bug. We now get the Teleporter message from the event logs, which contain the correct message nonce.